doc/rst/platforms/comm-layers/gasnet.rst

```diff
@@ -93,9 +92,10 @@ Current defaults are:
 ===================  ====================
 CHPL_COMM_SUBSTRATE  CHPL_GASNET_SEGMENT
 ===================  ====================
-aries                fast
 ibv                  large
+ofi                  fast (only when ``CHPL_TARGET_PLATFORM`` is ``hpe-cray-ex``)
```

Comment:
Unclear to me how to read the line above with respect to Omni-Path.  
For instance, does it mean Omni-Path will fall under "other" at the end of this table?

Reply:
Yes, the way the code is written today a non-hpe cray system with omni-path using gasnet will default to `everything`